include/types.h

```diff
@@ -2349,6 +2351,7 @@ typedef struct user_options
   bool         hex_salt;
   bool         hex_wordlist;
   bool         increment;
+  bool         increment_inverse;
```

Comment:
Isn't it better to use an `enum` type for it?

```suggestion
  increment_t increment;
```

It could be just like the following:

```c
typedef enum increment {
    INCREMENT_NONE,
    INCREMENT_NORMAL,
    INCREMENT_INVERSED,
} increment_t;
```

Reply:
Feels unnecessary imo but resolved:
https://github.com/hashcat/hashcat/pull/3960/commits/3db7f01dfa27cf7d1ea03c311be343a6bd4d9dd6